cargo-dist/src/config/mod.rs

```diff
@@ -18,6 +18,10 @@ use crate::{
     METADATA_DIST,
 };
 
+mod version;
+pub use version::ConfigVersion;
+pub use version::get_version;
```

Comment:
could collapse this into the above i think?

Reply:
Fixing that also fixed the failed `rust fmt` in CI, conveniently.